source/datetime.cpp

```diff
@@ -239,18 +240,20 @@ namespace nana
 			return days + d;
 		}
 
-		unsigned date::month_days(unsigned year, unsigned month)
+		unsigned date::month_days(const unsigned year, const unsigned month)
 		{
-			unsigned num[] = {31, 0, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};
-			if(month != 2)
-				return num[month - 1];
+			if (month != 2)
+			{
+				const std::array<unsigned, 12> days_in_month = { 31, 0, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31 };
```

Comment:
How about using `constexpr` insted of `const` here?

Reply:
Agreed, I have updated this patch with the change